package.json

```diff
@@ -10,7 +10,7 @@
     "serve": "serve dist",
     "start": "webpack-serve ./webpack.dev.js",
     "server": "node scripts/server",
-    "server:hot": "PG_CONNECTION_STRING=postgres://postgres:postgres@localhost:5432/postgres forever -w -t --killSignal=SIGTERM --watchDirectory scripts scripts/server.js",
+    "server:hot": "PG_CONNECTION_STRING=postgres://postgres:postgres@localhost:5678/postgres forever -w -t --killSignal=SIGTERM --watchDirectory scripts scripts/server.js",
```

Comment:
Changed the port to not conflict with my local jore-graphql DB. This command is for local dev only, so it does not impact anything else.